code/modules/mob/living/simple_animal/bees.dm

```diff
@@ -16,7 +16,7 @@
 	turns_per_move = 6
 	var/obj/machinery/portable_atmospherics/hydroponics/my_hydrotray
 
-/mob/living/simple_animal/bee/Initialize(loc, var/obj/machinery/apiary/new_parent)
+/mob/living/simple_animal/bee/Initialize(mapload, obj/machinery/apiary/new_parent)
```

Comment:
Bees are pretty broken

Reply:
They are. What do you suggest?

Reply:
just leave the code for it alone

Reply:
The change is positive, as it removes an error. The thing has further problems, but it's unrelated to this.
Changing it back is just extra work for no benefit.

Reply:
the code isnt used at all just leave it alone.

Reply:
I guess I'll remove it in a different PR, but the change is not negative.

Reply:
It'll conflict with the pr already up that removes them